### DISCUSSION TITLE
5 git jobs have timed out after git.timeout 400ms, disabling git integration.

### DISCUSSION THREAD

**Discussion by yk0n9** [nvim-tree/nvim-tree.lua#2737] [Q&A] [answered]
This prompt will appear when I open a large project.
Please tell me how should I deal with it? Or how to increase its limit?
I am using Lazy.nvim
![560ee33251e0f149f19fc2fabb9f4d0](https://github.com/nvim-tree/nvim-tree.lua/assets/66518156/42eeff2f-1720-4c3e-84e0-8bae45ecc218)

**Comment by alex-courtis** [comment, marked as answer]
See `:help nvim-tree.git.timeout`

See https://github.com/nvim-tree/nvim-tree.lua/wiki/Troubleshooting#git-timeouts

Run `time git --no-optional-locks status --porcelain=v1 --ignored=matching -u` to see how long git itself takes. The elapsed total (last) is the key metric.
